projects/dsp-ui/src/lib/viewer/values/color-value/color-value.component.scss

```diff
@@ -16,7 +16,7 @@
 }
 
 .color-preview {
-  border: 1px solid rgba(33,33,33.5);
+  border: 1px solid rgba(33,33,33,.5);
```

Comment:
.5 or 0.5??

Reply:
yes, in scss we don't need a leading 0

Reply:
https://github.com/sds/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero